mdsuite/calculators/green_kubo_ionic_conductivity.py

```diff
@@ -170,7 +170,7 @@ def _calculate_prefactor(self):
 
         """
         # Calculate the prefactor
-        numerator = (elementary_charge ** 2) * (self.experiment.units["length"] ** 2)
+        numerator = (elementary_charge**2) * (self.experiment.units["length"] ** 2)
```

Comment:
And why it isn't here? Just curious

Reply:
I have no idea - funny catch. I have to admit I did not go through the changed files by black.